sdk/formrecognizer/Azure.AI.FormRecognizer/src/ExtractedField.cs

```diff
@@ -4,55 +4,63 @@
 using System.Collections.Generic;
 using System.Globalization;
 using System.Linq;
+using Azure.AI.FormRecognizer.Models;
 
-namespace Azure.AI.FormRecognizer.Models
+namespace Azure.AI.FormRecognizer.Custom
 {
+    // Maps to KeyValuePair
+
+    /// <summary>
+    /// </summary>
     public class ExtractedField
     {
         internal ExtractedField(KeyValuePair_internal field, ReadResult_internal readResult)
         {
-            // Unsupervised
             Confidence = field.Confidence;
-            Label = field.Key.Text;
 
-            LabelBoundingBox = field.Key.BoundingBox == null ? null : new BoundingBox(field.Key.BoundingBox);
+            Name = field.Key.Text;
+            NameBoundingBox = field.Key.BoundingBox == null ? null : new BoundingBox(field.Key.BoundingBox);
+
             if (field.Key.Elements != null)
             {
-                LabelRawExtractedItems = ConvertTextReferences(readResult, field.Key.Elements);
+                NameRawExtractedItems = ConvertTextReferences(readResult, field.Key.Elements);
             }
 
             Value = field.Value.Text;
-            ValueBoundingBox = new BoundingBox(field.Value.BoundingBox);
+            ValueBoundingBox = field.Value.BoundingBox == null ? null : new BoundingBox(field.Value.BoundingBox);
 
             if (field.Value.Elements != null)
             {
                 ValueRawExtractedItems = ConvertTextReferences(readResult, field.Value.Elements);
             }
         }
 
-        internal ExtractedField(KeyValuePair<string, FieldValue_internal> field)
-        {
-            // Supervised
-            Confidence = field.Value.Confidence;
-            Label = field.Key;
-            Value = field.Value.Text;
-            ValueBoundingBox = new BoundingBox(field.Value.BoundingBox);
-        }
+        /// <summary>
+        /// </summary>
+        public float Confidence { get; internal set; }
+
+        /// <summary>
+        /// </summary>
+        public string Name { get; internal set; }
 
-        // TODO: Why can this be nullable on FieldValue.Confidence?
-        // https://github.com/Azure/azure-sdk-for-net/issues/10378
-        public float? Confidence { get; internal set; }
-        public string Label { get; internal set; }
+        /// <summary>
+        /// </summary>
+        public BoundingBox NameBoundingBox { get; internal set; }
 
-        // TODO: Make this nullable to indicate that this is an optional field.
-        // https://github.com/Azure/azure-sdk-for-net/issues/10361
-        // Not currently supported for Track2 libraries.
-        public BoundingBox LabelBoundingBox { get; internal set; }
+        /// <summary>
+        /// </summary>
+        public IReadOnlyList<RawExtractedItem> NameRawExtractedItems { get; internal set; }
```

Comment:
Is this the equivalent of elements?

Reply:
It is.  We have an issue to rename them to be consistent with the other languages.  https://github.com/Azure/azure-sdk-for-net/issues/10623